src/Module/Item/Display.php

```diff
@@ -162,6 +162,8 @@ protected function content(array $request = []): string
 
 		$output .= $this->getDisplayData($item);
 
+		header('Vary: Accept', false);
```

Comment:
`/display` doesn't output anything other than HTML, when an `Accept: application/json` header is received, there's a redirection to `/objects`, is the `Vary` header still warranted?

Reply:
I think it is still warranted. A caching layer still needs to know the server would have responded with a redirect instead of content if a request comes in with the wrong `Accept` header and a content response was cached.

(It's more likely to happen with the redirect from `/objects` to `/display` than here, though; and leaving aside whether the content should have been cached)

I have moved the `header()` call to the beginning of processing, so it is similar to the other modules.